src/navigation/NavigationTypes.ts

```diff
@@ -9,19 +9,17 @@ export interface DocumentResult {
   numberOfTries: number
 }
 
-interface ExerciseParams {
+export interface ExerciseParams {
   disciplineTitle: string
   documents: Document[]
   closeExerciseAction: CommonNavigationAction
 }
 
-export type ExercisesParams =
-  | {
-      documents: null
-      discipline: Discipline
-      disciplineTitle: string
-    }
-  | ExerciseParams
+export interface ExercisesParams {
```

Comment:
If you change this type please change the logic in `ExercisesScreen` as well with the suggestion below. Also the types should be cleaned up then: Remove `documents: null` and `disciplineTitle`. Otherwise leave it as it was and I will clean it up later.

Reply:
Done in https://github.com/digitalfabrik/lunes-app/pull/287/commits/13c7cf9bbed09646973c9eac373532f95d6987e2

Reply:
It think it should still be possible to navigate to ExerciseScreen with passing documents instead of discipline, but I can add this later if wanted.